mocks/auth.js

```diff
@@ -35,6 +38,15 @@ class FakeAuth {
     return Promise.resolve(this.currentUserRecord);
   }
 
+  getUser() {
+    return Promise.resolve(mockGetUser(...arguments));
+  }
+
+  setCustomUserClaims() {
+    mockSetCustomUserClaims(...arguments);
+    return Promise.resolve({});
```

Comment:
Similar to `getUser` could we have this return `mockSetCustomUserClaims(...arguments)` or an empty object, that way a tester could do `mockSetCustomUserClaims.mockReturnValue` in their own test if they needed to explicitly stub something?